src/urdf.rs

```diff
@@ -63,29 +63,27 @@ pub fn add_geometry(
         }
         urdf_rs::Geometry::Mesh {
             ref filename,
-            scale,
+            ref scale,
         } => {
-            let scale = scale.unwrap_or(DEFAULT_MESH_SCALE);
+            let scale = scale.clone().unwrap_or(DEFAULT_MESH_SCALE);
```

Comment:
Clones in this PR will become unneeded once https://github.com/openrr/urdf-rs/pull/71 is merged.